prow/moonraker/client.go

```diff
@@ -48,13 +49,14 @@ type Client struct {
 	host        string
 	httpClient  *http.Client
 	configAgent prowConfigAgentClient
+	sync.Mutex
```

Comment:
Typically place sync.Mutex above the fields it protects, so here, e.g.

....
configAgent

sync.Mutex  // protects below
httpClient